examples/circle_packing.py

```diff
@@ -21,11 +21,10 @@
         t = cp.Variable()
         diff_vars.append(ncvx.Annulus(2, 0.5 * (r[i]+r[j]), N))
         constraints += [
-            cp.vstack(x_vals[i] - x_vals[j],
-                      y_vals[i] - y_vals[j]) == diff_vars[-1]]
+            cp.vstack((x_vals[i] - x_vals[j], y_vals[i] - y_vals[j])) == diff_vars[-1]]
 
 prob = cp.Problem(objective, constraints)
-result = prob.solve(method="relax-round-polish", polish_depth=100)
+result = prob.solve(method="relax-round-polish", polish_depth=100, verbose=True)
```

Comment:
could you remove the verbose keyword here?

Reply:
I removed the verbose